src/main/java/edu/kis/vh/nursery/HanoiRhymer.java

```diff
@@ -1,13 +1,27 @@
 package edu.kis.vh.nursery;
 
+/**
+ * Simple Hanoi Rhyme.
+ */
 public class HanoiRhymer extends DefaultCountingOutRhymer {
 
-    int totalRejected = 0;
+    private static final int NUMBER_OF_REJECTED_RHYMES = 0;
```

Comment:
totalRejected represents NUMBER_OF_REJECTED_RHYMES, but this is INITIAL value. This is a plain 0 (I would not consider it a constant)